demo/src/app.jsx

```diff
@@ -912,10 +911,13 @@ function App() {
                 locale={computedLanguage}
                 messages={messages[computedLanguage]}
             >
-                <AppContent
-                    language={language}
-                    onLanguageClick={handleLanguageClick}
-                />
+                <Websocket urls={{ [WS_URL_KEY]: WS_URL }}>
```

Comment:
What happens if we use the `useListener` without using HOC `Websocket` somewere in the hierarchy ?
Seems like they are linked component/hook.

Reply:
yes there are linked as context Providers and useContext are.

Reply:
f you use `useContext` without the provider, an error is thrown indicating that the context is not found.